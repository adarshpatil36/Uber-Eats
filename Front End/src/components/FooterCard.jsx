import React from "react";

export default function FooterCard() {
  return (
    <footer className="footerCard">
      <span>Â© 2020 Uber Technologies, Inc.</span>
      <span>Privacy Policy | Terms of Use</span>
    </footer>
  );
}
