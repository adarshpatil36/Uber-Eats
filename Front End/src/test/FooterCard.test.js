import React from "react";
import { cleanup, fireEvent, render } from "@testing-library/react";
import ReactDOM from "react-dom";
import FooterCard from "../components/FooterCard";

import renderer from "react-test-renderer";
it("renders FooterCard without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<FooterCard></FooterCard>, div);
});

it("renders FooterCard with proper label", () => {
  const { queryByLabelText } = render(<FooterCard></FooterCard>);
  expect(queryByLabelText).toBeTruthy();
});

it("renders correctly", () => {
  const tree = renderer.create(<FooterCard></FooterCard>).toJSON();
  expect(tree).toMatchSnapshot();
});
