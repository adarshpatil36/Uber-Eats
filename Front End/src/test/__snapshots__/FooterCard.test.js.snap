// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="footerCard"
  data-testid="footerCard"
>
  <span>
    Â© 2020 Uber Technologies, Inc.
  </span>
  <span>
    Privacy Policy | Terms of Use
  </span>
</footer>
`;
